examples/build.gradle

```diff
@@ -29,6 +29,7 @@ dependencies {
     implementation "io.grpc:grpc-protobuf:${grpcVersion}"
     implementation "io.grpc:grpc-stub:${grpcVersion}"
     compileOnly "javax.annotation:javax.annotation-api:1.2"
+    compile "org.apache.logging.log4j:log4j-api:2.11.2"
```

Comment:
We've been putting examples that require additional dependencies in their own project.

Reply:
examples-log4j okay?

Reply:
SGTM

Reply:
Done.